mlir/lib/Dialect/ArmSVE/Transforms/LegalizeForLLVMExport.cpp

```diff
@@ -192,6 +194,7 @@ void mlir::populateArmSVELegalizeForLLVMExportPatterns(
                SmmlaOpLowering,
                UdotOpLowering,
                UmmlaOpLowering,
+               DupQLaneLowering,
```

Comment:
Let's just order this alphabetically. You can add a note in the summary that that's an additional NFC in this PR.

This could be a separate PR, but the noise level will be low, so OK to do it here.